lib/processLayerStyles.js

```diff
@@ -28,10 +29,29 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
             style: []
         }
         addOptionalProperty("background", data.style, layerFillWithShadows(layerStyle, outputFolder, sketchFile) )
-        addOptionalProperty("border", data.style, layerBorder(layerStyle) )
         addOptionalProperty("blur", data.style, layerBlur(layerStyle) )
-        // Apply the corner radius modifier
-        addOptionalProperty("cornerRadius", data.style, cornerRadius(layerStyle, data.name, modifiers.radius) )
+
+        // take the style, if there is a matching corner radius apply it.
+        let currentModifier = {modifier: null, score: 0};
```

Comment:
this chunk is big enough it should go in it's own function/method

Reply:
Done 👍